app/views/candidate_interface/application_choices/_guidance.html.erb

```diff
@@ -1,4 +1,3 @@
-<p class="govuk-body">You can apply for up to 3 courses at this stage of your application.</p>
-<p class="govuk-body">Not all courses and training providers are signed up to this service. If you choose a course that isn’t signed up to <%= service_name %>, you’ll be directed to UCAS to continue your application.</p>
-<p class="govuk-body">You can preview <%= govuk_link_to 'courses currently available', candidate_interface_providers_path %> on <%= service_name %>.</p>
-<p class="govuk-body">Get support by emailing <%= govuk_link_to 'becomingateacher@digital.education.gov.uk', 'mailto:becomingateacher@digital.education.gov.uk' %>.</p>
+<p class="govuk-body">You can apply for up to 3 courses.</p>
+<p class="govuk-body">Some courses are not on <%= service_name %> yet and are only available through UCAS Teacher Training.</p>
+<p class="govuk-body"><%= govuk_link_to 'Check which courses you can apply to on ' + service_name, candidate_interface_providers_path %></p>
```

Comment:
```suggestion
<p class="govuk-body"><%= govuk_link_to 'Check which courses you can apply to on ' + service_name, candidate_interface_providers_path %>.</p>
```

Should this have a full stop? I'm actually not sure haha

Reply:
I think not

Reply:
I did think but was kinda following the design so wasn't sure will update now and then 🚢

Reply:
Just to clarify I haven't added the full stop, my bad for the confusion

Reply:
Hey @george-bit @tvararu @davidgisbey, my rule of thumb is if the whole sentence is hyperlinked and it's an instruction, such as 'Learn more.....' 'Check.....', 'Go to....' - don't use a full stop. That's based on what I've seen elsewhere on gov.uk